cumulus/ansible/tasks/cluster.py

```diff
@@ -90,3 +75,46 @@ def run_ansible(cluster, profile, secret_key, extra_vars,
 
         r = requests.patch(status_url, headers=headers, json=updates)
         check_status(r)
+
+
+@command.task
+def provision_cluster(playbook, cluster, profile, secret_key, extra_vars,
+                      girder_token, log_write_url, post_status):
+    playbook = get_playbook_path(playbook)
+    playbook_variables = get_playbook_variables(cluster, profile, extra_vars)
+
+    env = os.environ.copy()
+    env.update({"AWS_ACCESS_KEY_ID": profile['accessKeyId'],
+                "AWS_SECRET_ACCESS_KEY": secret_key,
+                "GIRDER_TOKEN": girder_token,
+                "LOG_WRITE_URL": log_write_url,
+                "CLUSTER_ID": cluster["_id"]})
+
+    inventory = os.path.join(os.path.dirname(__file__), 'dynamic-inventory')
+
+    run_playbook(playbook, inventory, playbook_variables,
+                 env=env, verbose=3)
+
+    check_girder_cluster_status(cluster, girder_token, post_status)
+
+
+@command.task
+def run_ansible(playbook, cluster, profile, secret_key, extra_vars,
+                girder_token, log_write_url, post_status):
+    playbook = get_playbook_path(playbook)
+    playbook_variables = get_playbook_variables(cluster, profile, extra_vars)
+
+    env = os.environ.copy()
+    env.update({"AWS_ACCESS_KEY_ID": profile['accessKeyId'],
+                "AWS_SECRET_ACCESS_KEY": secret_key,
+                "GIRDER_TOKEN": girder_token,
+                "LOG_WRITE_URL": log_write_url,
+                "CLUSTER_ID": cluster["_id"]})
+
```

Comment:
Sorry,  this got fixed,  but had to be un-fixed to resolve conflicts when merging in master (with the gpl-refactor branch). I thought we re-fixed it,  but apparently not :confounded: